databases/backends/postgres.py

```diff
@@ -216,14 +216,12 @@ async def execute(self, query: ClauseElement) -> typing.Any:
         query_str, args, result_columns = self._compile(query)
         return await self._connection.fetchval(query_str, *args)
 
-    async def execute_many(self, queries: typing.List[ClauseElement]) -> None:
+    async def execute_many(
+        self, queries: typing.List[ClauseElement], values: typing.List[dict]
+    ) -> None:
```

Comment:
I presume this it not a breaking change for users because this is an internal method and `values` is always provided by us in the public method?

Reply:
I'm not sure I understand what you mean here, but I think users are only supposed to use execute_many from the core file, not from any of the backends

Reply:
Yep that's the question — I want to gauge if changing this function signature is a breaking change.

Reply:
yea I don't think it should break anything